conductor/src/main.rs

```diff
@@ -549,61 +549,69 @@ async fn run(metrics: CustomMetrics) -> Result<(), ConductorError> {
                 info!("{}: Deleting namespace {}", read_msg.msg_id, &namespace);
                 delete_namespace(client.clone(), &namespace).await?;
 
-                if is_cloud_formation {
-                    info!("{}: Deleting cloudformation stack", read_msg.msg_id);
-                    delete_cloudformation(aws_region.clone(), &namespace).await?;
-                }
+                if read_msg.message.event_type == Event::Delete {
```

Comment:
IMHO this if statement isn't needed.  If the instance is paused, we have already removed the IAM/Workload Idenitity/UAMI from the cloud provider anyway.  It's safe to run for both events (`Delete` and `ScheduledDeletion`).